src/terra-dev-site/contributing/Conventions.d.contributing.mdx

```diff
@@ -20,6 +21,90 @@ import PairProgrammingIcon from '../icon/PairProgrammingIcon';
 </div>
 </div>
 
+### CSS Modules
+
+We use a concept called [CSS Modules](https://github.com/css-modules/css-modules) when creating the styles for the Terra UI components. A CSS Module is a CSS file, SCSS file in our case, in which all class names and animation names are scoped locally by default.
```

Comment:
For Terra we default css modules scope to be global:
https://github.com/cerner/terra-toolkit/blob/master/config/webpack/webpack.config.js#L64

Reply:
Added some updates on this in this commit: https://github.com/cerner/terra-ui/pull/230/commits/6d312fa09f5f0394874c9d0e8ac46c5599310822